COMMUNICATION_MODE_SYNC.md

```diff
@@ -0,0 +1,224 @@
+# Communication Mode Synchronization
```

Comment:
Do we want to keep this file?

Reply:
not necessarily no... i'm still working through how to deploy a working stack without knative eventing & kafka ... its close